packages/terra-core-docs/src/terra-dev-site/doc/image/example/DecorativeImage/DecorativeImageVaraints.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { DecorativeImage } from 'terra-image';
+import decorativeImage from '../../assets/crocus.jpg';
+import placeholderPic1000x200 from '../../assets/1000x200.png';
+
+const DecorativeImageVaraints = () => (
```

Comment:
This file's name and this variable name "Varaints" -> "Variants"

Reply:
[changed](https://github.com/cerner/terra-core/pull/3585/commits/a72252e81ed3e48fbebc3ff03878cf6f428b0261#diff-bd0e9eec7cf28f31639dd12deaeb4b655703211779967016bd20c7a3274ed2cc)